src/objects/core/zcl_abapgit_serialize.clas.abap

```diff
@@ -303,63 +299,81 @@ CLASS zcl_abapgit_serialize IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD determine_max_threads.
+  METHOD determine_max_processes.
+    DATA: lo_settings TYPE REF TO zcl_abapgit_settings,
+          li_exit     TYPE REF TO zif_abapgit_exit.
 
     IF iv_force_sequential = abap_true.
-      rv_threads = 1.
+      rv_processes = 1.
       RETURN.
     ENDIF.
 
-    IF gv_max_threads >= 1.
-* SPBT_INITIALIZE gives error PBT_ENV_ALREADY_INITIALIZED if called
-* multiple times in same session
-      rv_threads = gv_max_threads.
-      RETURN.
+    IF gv_max_processes IS INITIAL.
+      lo_settings = zcl_abapgit_persist_factory=>get_settings( )->read( ).
+
+      IF zcl_abapgit_factory=>get_environment( )->is_merged( ) = abap_true
```

Comment:
the check here means that we can skip the "function exists" check below (if the function really does not exist in the dev version, then causing a dump is good).

Reply:
I agree and removed the call. The transpiler setup doesn't have the function module either though it seems ;)